ledger/ledger-on-memory/src/app/scala/com/daml/ledger/on/memory/Main.scala

```diff
@@ -96,6 +113,25 @@ object Main {
               )
             )
         }
+      parser
+        .opt[Boolean]("always-pre-execute")
+        .optional()
+        .text(
+          "Whether to enable pre-execution (mutually exclusive with batching and will disable it).")
```

Comment:
```suggestion
          "Enable pre-execution (mutually exclusive with batching)")
```

Reply:
The flag will actually force pre-execution so I used a slight variation on your proposal (and changed the flag name itself).